src/shared/lib/oql/AccessorsForOqlFilter.ts

```diff
@@ -123,7 +127,13 @@ export default class AccessorsForOqlFilter
     }
 
     public gene(d: Datum) {
-        return d.gene.hugoGeneSymbol;
+        if (
+            this.molecularAlterationType(d.molecularProfileId) ===
+            AlterationTypeConstants.STRUCTURAL_VARIANT
+        ) {
+            return (d as StructuralVariant).site1HugoSymbol;
+        }
+        return (d as any).gene.hugoGeneSymbol;
```

Comment:
why does it now have to be d as any?  seems like typing should be same for this case?

Reply:
`gene` object doesn't exist in `StructuralVariant` and `AnnotatedStructuralVariant` types.